garage/experiment/local_tf_runner.py

```diff
@@ -210,45 +293,95 @@ def log_diagnostics(self, pause_for_plot=False):
 
     def train(self,
               n_epochs,
+              batch_size,
               n_epoch_cycles=1,
-              batch_size=None,
               plot=False,
               store_paths=False,
-              pause_for_plot=False):
+              pause_for_plot=False,
+              **kwargs):
         """Start training.
 
         Args:
-            n_epochs: Number of epochs.
-            n_epoch_cycles: Number of batches of samples in each epoch.
+            n_epochs(int): Number of epochs.
+            batch_size(int): Number of environment steps in one batch.
+            n_epoch_cycles(int): Number of batches of samples in each epoch.
+                This is only useful for off-policy algorithm.
+                For on-policy algorithm this value should always be 1.
+            plot(bool): Visualize policy by doing rollout after each epoch.
+            store_paths(bool): Save paths in snapshot.
+            pause_for_plot(bool): Pause for plot.
+
+        Returns:
+            The average return in last epoch cycle.
+
+        """
+        return self._train(
+            n_epochs=n_epochs,
+            n_epoch_cycles=n_epoch_cycles,
+            batch_size=batch_size,
+            plot=plot,
+            store_paths=store_paths,
+            pause_for_plot=pause_for_plot,
+            **kwargs)
+
+    def resume(self):
```

Comment:
if  you want to modify the training arguments in resume, i think resume should have the same function signature as train (but its default arguments are the ones from the first call to train)

Reply:
What about now. It should look much better.